test/testenv/verificationutils.go

```diff
@@ -769,6 +769,28 @@ func VerifyAppListPhase(deployment *Deployment, testenvInstance *TestEnv, name s
 	}
 }
 
+// VerifyAppInstallInProgress verify given app installation is in progress, i.e when Status is between 101 and 303
+func VerifyAppInstallInProgress(deployment *Deployment, testenvInstance *TestEnv, name string, crKind string, appSourceName string, appList []string) {
+	for _, appName := range appList {
+		gomega.Eventually(func() enterpriseApi.AppPhaseStatusType {
+			appDeploymentInfo, _ := GetAppDeploymentInfo(deployment, testenvInstance, name, crKind, appSourceName, appName)
+			return appDeploymentInfo.PhaseInfo.Status
+		}).Should(gomega.BeNumerically("~", 303, 203)) //Check status value is between 100 and 303
+	}
+}
+
+// WaitForAppInstall waits until an app is correctly installed (having status equal to 303)
+func WaitForAppInstall(deployment *Deployment, testenvInstance *TestEnv, name string, crKind string, appSourceName string, appList []string) {
+	var status enterpriseApi.AppPhaseStatusType = 303
```

Comment:
Since the phase is declared as a const in pkg. Can we import it here?

Reply:
AppPkgInstallComplete is a const int, it can't be directly compared to an enterpriseApi.AppPhaseStatusType actually